src/main/java/org/traccar/protocol/WatchProtocolDecoder.java

```diff
@@ -266,6 +266,10 @@ protected Object decode(
 
             sendResponse(channel, id, index, "TKQ");
 
+        } else if (type.equals("TKQ2")) {
```

Comment:
Can we please combine TKQ and TKQ2 cases.

Reply:
I commited the combined condition for these cases.